cg/server/app_config.py

```diff
@@ -4,23 +4,25 @@
 class AppConfig(BaseSettings):
```

Comment:
Can we add a doc string to not confuse people with the dummy values?
```suggestion
class AppConfig(BaseSettings):
"""Default values are overridden by environment variable at run-time."""
```